webui/react/src/pages/Clusters.tsx

```diff
@@ -38,17 +36,9 @@ const Clusters: React.FC = () => {
   const navigate = useNavigate();
 
   const [tabKey, setTabKey] = useState<TabType>(tab || DEFAULT_TAB_KEY);
-  const loadableResourcePools = useResourcePools();
-  const resourcePools = Loadable.getOrElse([], loadableResourcePools); // TODO show spinner when this is loading
-  const overview = useClusterOverview();
-  const agents = useAgents();
-
-  const cluster = useMemo(() => {
-    return Loadable.match(Loadable.all([agents, overview]), {
-      Loaded: ([agents, overview]) => clusterStatusText(overview, resourcePools, agents),
-      NotLoaded: () => undefined, // TODO show spinner when this is loading
-    });
-  }, [overview, resourcePools, agents]);
+
+  useRefetchClusterData();
+  const clusterStatus = useClusterStatus();
```

Comment:
This is a case where useValueMemoizedObservable would be very useful since we're polling and so can get many unecessary re-renders.

Reply:
ok, i would use it in the actual `useClusterStatus` hook though right?

Reply:
Sure.

Reply:
ended up checking for `isEqual` in the fetch functions themselves. think this should serve the same purpose.